### ISSUE TITLE
how to create MDI forms with this?

### ISSUE THREAD

**Issue by Shadowblitz16** [andlabs/libui#475]
how would I create MDI forms with this library?

**Comment by bcampbell** [comment]
I'm pretty sure there's no support for MDI in libui.
I can't imagine there are any plans to implement it - it'd be a bit of a cross-platform nightmare (eg on OSX).

**Comment by Shadowblitz16** [comment]
that's too bad it would be good for game editors

**Comment by andlabs** [comment]
Yes, that's a work in progress. In the meantime you can open multiple windows.

**Comment by Shadowblitz16** [comment]
:D

**Comment by rubyFeedback** [comment]
MDI was quite difficult to do in ruby-gtk (thus gtk+) as well for some time. I remember the year
2006 still, still having the old mdi.rb file in the pre-notebook days (once Gtk::Notebook existed,
it was easier, but we had to do type_register() and twiddle around with things that were not
so simple to understand).. :-)
